examples/log_sig_example.py

```diff
@@ -26,4 +26,6 @@
 
 # send them over network
 # Logicsig will reject the transaction
-acl.send_transaction(lstx)
+
+# TODO: as of v2.0.0, this will fail.
```

Comment:
Discovered to be broken according to `mypy`

Reply:
Does mypy complain that the output is not consumed? Sending the logicsig seems integral to this example

Reply:
Sure thing, I'm not even sure if we enforce `mypy` to be run on the `examples` yet, so we could uncomment back out and investigate later.

Reply:
It looks like we only `mypy` algosdk: https://github.com/algorand/py-algorand-sdk/blob/develop/Makefile#L14